Assets/RedBlueGames/MulliganRenamer/Tests/Editor/LocaleTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿/* MIT License
+
+Copyright (c) 2020 Murillo Pugliesi Lopes, https://github.com/Mukarillo
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
+
+namespace RedBlueGames.MulliganRenamer
+{
+    using NUnit.Framework;
+    using System;
+
+    public class LocaleTests
+    {
+        [Test]
+        public void CheckAllLanguageKeys_WithEnglish()
+        {
+            var englishLanguage = LocaleManager.Instance.AllLanguages.Find(x => x.LanguageKey == "en");
```

Comment:
Another valid approach to this would be to cut out LocaleManager entirely and simply verify the files with `JSONUtility.FromJson`.

If you go that route, which I'd probably say is the "Best" because it's more clearly a "Unit" test (testing only one thing), you'd probably want to spin up a 2nd test file for the ChangeLanguage method because that one is truly LocaleManager functionality that's being tested.